torchaudio/models/wavernn.py

```diff
@@ -1,18 +1,41 @@
-from typing import List, Tuple
+from typing import List, Tuple, Dict, Any
 
 import torch
 from torch import Tensor
 from torch import nn
 
+from ._utils import load_state_dict_from_url
+
+
 __all__ = [
     "ResBlock",
     "MelResNet",
     "Stretch2d",
     "UpsampleNetwork",
     "WaveRNN",
+    "get_pretrained_wavernn",
 ]
 
 
+model_config_and_urls: Dict[str, Tuple[str, Dict[str, Any]]] = {
```

Comment:
Since this is a private global object, please prefix it with underscore and capitalize all.

Reply:
Thanks for the comment.
It is fixed [here](https://github.com/pytorch/audio/pull/1612/files#diff-7f4ad259da182d3840fe4c75dd0c0b7b9fe30f31503df0eff87b1e5c9378cac4R22)